### DISCUSSION TITLE
How Much of the Processes of BN Are Now Part of Zotero Out of the Box?

### DISCUSSION THREAD

**Discussion by kertripmerth** [windingwind/zotero-better-notes#749] [General]
Hello,

First, I want to congratulate you, windingwind, for all the **AMAZING** innovations you've contributed to Zotero.

When I see how you are consistently looking so far over the horizon and seeing things that even the developers themselves can't envision, I feel impressed.  Thank you, windingwind!

After having gone back and forth between several different citation managers, I've come back to Zotero.  I'm just really digging in now.

Please can you help me with one thing?  It seems like most of the main functionalities that you built into better notes are now part of zotero out of the box.

In your opinion, what are the main key functional differences and advantages of better notes compared to the latest iteration of zotero?  

In what ways are they very similar, but what are the standout differences that make better note the thing to have?

Sincerely,

Kert

**Comment by windingwind** [comment]
Zotero already has a note system, in which you can write down things, export to markdown (although without images in binary), and import from highlights. If these can satisfy you, then you don't need BN.

> what are the main key functional differences and advantages of better notes compared to the latest iteration of zotero?

What BN provides are listed [here](https://github.com/windingwind/zotero-better-notes#-what-can-it-do). Most people come to BN for the note link, note-markdown export/syncing, and note template. None of these are built-in in Zotero.

> In what ways are they very similar, but what are the standout differences that make better note the thing to have?

It depends on what kind of note-taking system you are looking for. If you just want a place to write down ideas while reading, then BN makes little difference. If you want to do systematically organizing, comparing, and summarizing, then this is what you have to do manually without BN and you can do it automatically (if set up properly) with BN.

Use cases:

1. Collect ideas from multiple PDFs, get them sorted by your desired order (annotation color/paper title), and do a comparison in one note.
2. Take notes inside Zotero while reading and synchronize notes with 3rd-party note managers, e.g. Obsidian.
3. [Zettelkasten Method](https://zettelkasten.de/introduction/) in Zotero

**Reply by kertripmerth** [reply]
Hello windinwind,

**Ah!  BLESS you for your kind and detailed reply!!! :))))))))))))))!!!**

You wrote,

"Collect ideas from multiple PDFs, get them sorted by your desired order (annotation color/paper title), and do a comparison in one note."

That sounds VERY wonderful to me.  Is there any way you could make a few screen shots to illustrate how that works?

I'll tell you, when I read "Collect ideas from multiple PDFs, get them sorted by your desired order (annotation color/paper title), and do a comparison in one note.", I really want to shout with joy "YES!", as that is EXACTLY what I need.  

I'm already facing such a stack of PDFs in the research topic I'm doing.  

I want to use BN, but I feel worried because there's just not enough in terms of the documentation for me to understand how it works.

I have an idea.  Tell me what you think.  

There's a popular children's story called Treasure Island.  It is nominally considered to be "literature", primarily on the strength of the author's style, but the story is little less than a Disney pirate tale.  And, quite a fun one at that! :D

Below are three "resources" for Treasure Island (just wikipedia-like pages).  Please could you take a few quotes from each one and use them to illustrate a best use of BN?  The idea of, "...systematically organizing, comparing, and summarizing..." is pretty much exactly what I know I need to be able to do.  You've really put your finger on what is the bane of every student and research here in the early 21st century - how to keep from having just a scramble of ideas saved across 100 different directories and applications.

Please if you could take these three resources, pulling a quote or two from each one, to show how BN accomplishes its goal, that would be a HUGE help.

Nothing fancy.  Just screenshots and text.

Again, I feel VERY grateful to you for your SUSTAINED and AMAZING innovations to Zotero.  And, as a final note, for having just about **the COOLEST username** EVER! :)))

https://en.wikipedia.org/wiki/Treasure_Island
https://www.britannica.com/topic/Treasure-Island
https://www.neh.gov/humanities/2015/julyaugust/feature/treasure-island-author-robert-louis-stevenson-was-sickly-man-robu

Sincerely,

Kert

**Reply by windingwind** [reply]
## Results

collect annotations by color(sort by color):

![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/f893ab72-7139-4061-a084-0e5a2c2c27c4)

> You may change the title `red annotation` to whatever you want, like `summary` or `detials`

## Steps to setup

1. Goto https://github.com/windingwind/zotero-better-notes/discussions/227 and copy the content under `Template Code for Importing`
![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/cf7777df-1a53-4379-90a0-483f1c506c7f)

2. Open Zotero (with BN latest beta, 1.1.4-25 installed) and create annotations on attachments (PDF/EPUB/snapshot)
3. Menu-Edit-New Template from Clipboard
![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/2231104f-9fa5-48a1-9c27-a383ee890c4d)

## Steps to run the template

1. Select items in the library (or select manually later) and create a note from the template you just imported
![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/f94f7c86-95c9-4003-8812-d329069d49f7)

![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/a4eb66bd-c6b2-4af6-b5dc-aaeae97619a9)

2. You get the note.

**Reply by windingwind** [reply]
Sorry I haven't had time to read the story (i will), i just made an example to help you understand how it works

**Comment by kertripmerth** [comment]
😄 !!
